src/modules/core/components/CountDownTimer/CountDownTimer.css

```diff
@@ -0,0 +1,10 @@
+.container {
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+  color: color-mod(var(--temp-grey-blue-7) alpha(85%));
+}
+
+.time {
+  color: var(--dark);
+  opacity: 100%;
```

Comment:
I think the time would look better and closer to the design specs if the opacity was `85%`

Reply:
Good catch!